doc/Uplink.xml

```diff
@@ -331,38 +331,75 @@
-            validated. </para>
-        </listitem>
-        <listitem>
-          <para>AuthorizationServer :  JWTConfiguration token referring to an Authorization server
-            that provides JWT token to authorize with the uplink server.</para>
-        </listitem>
-        <listitem>
-          <para>Error : Optional user readable error information (readonly).</para>
-        </listitem>
-      </itemizedlist>
+      <table xml:id="table_uplink-config-parameters">
+        <title>Parameters description</title>
+        <tgroup cols="2">
+          <colspec colname="c1" colwidth="30*"/>
+          <colspec colname="c2" colwidth="70*"/>
+          <thead>
+            <row>
+              <entry>
+                <para>Parameter</para>
+              </entry>
+              <entry>Description</entry>
+            </row>
+          </thead>
+          <tbody valign="top">
+            <row>
+              <entry>RemoteAddress</entry>
+              <entry>
+                <para>Uniform resource locator by which the remote client can be reached.</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para>CertificateID</para>
+              </entry>
+              <entry>
+                <para>ID of the certificate to be used for client authentication, mandatory to
+                  configure when authentication mode is mTLS.</para>
+              </entry>
+            </row>
+            <row>
+              <entry>UserLevel</entry>
+              <entry>
+                <para>Authorization level that will be assigned to the uplink connection.</para>
+              </entry>
+            </row>
+            <row>
+              <entry>Status</entry>
+              <entry>
+                <para>Current connection status</para>
+              </entry>
+            </row>
+            <row>
+              <entry>CertPathValidationPolicyID</entry>
+              <entry>
+                <para>CertPathValidationPolicyID used to validate the uplink server certificate. If
+                  not configured, server certificate validation behavior is undefined and the device
+                  may either apply a vendor specific default validation policy or skip validation at
+                  all. </para>
+              </entry>
+            </row>
+            <row>
+              <entry>AuthorizationServer</entry>
+              <entry>
+                <para>JWTConfiguration token referring to an authorization server that provides JWT
+                  token to authorize with the uplink server, mandatory to configure when the
+                  authentication mode is JWT.</para>
+              </entry>
+            </row>
+            <row>
+              <entry>Error</entry>
+              <entry>
+                <para>Optional user readable error information (readonly).</para>
+              </entry>
+            </row>
+          </tbody>
+        </tgroup>
+      </table>
```

Comment:
Tying the requirements to a non-existing current authentication mode parameter doesn't make much sense. Propose to keep original list layout and add a requirements section that describe how to configure when using mTLS and one for JWT Configuration.

Reply:
"tying the requirements to a non-existing current authentication mode parameter doesn't make much sense. "
 - I meant it as based on "authorization mode" capability, not mode which I can correct.

Ref Section 6.5 capabilities
- AuthorizationModes
  - Supported authorization mode [mTLS, JWT]

Reply:
The way I understand Hans' comment is that the new wording reads as if the AuthenticationMode is a parameter in the configuration, which is a bit confusing since it isn't.
```diff
- mandatory to configure when the authentication mode is JWT.
+ when configured, the device will use the JWT authentication mode.
```
Might be a better wording?